x-pack/test/functional/page_objects/space_selector_page.js

```diff
@@ -52,6 +52,27 @@ export function SpaceSelectorPageProvider({ getService, getPageObjects }) {
         await PageObjects.common.sleep(1000);
       });
     }
+
+    /**
+     * @param {string} [spaceId]
+     * @param {string} [opts] - options object
+     * @param {string} [opts.redirectPath] - subroute to redirect to (the part after the hash #)
+     */
+    async switchToSpace(spaceId = 'default', opts = { redirectPath: null }) {
+      log.debug(`switchToSpace(${spaceId})`);
+      await this.openSpacesNav();
+      await this.clickSpaceAvatar(spaceId);
+      if (opts.redirectPath) {
+        const serverBasePath = spaceId === 'default'
+          ? PageObjects.common.getHostPort() + '/app/kibana'
+          : PageObjects.common.getHostPort() + `/s/${spaceId}/app/kibana`;
+        await retry.try(async () => {
+          log.debug(`switchToSpace(${spaceId}): redirect to ${serverBasePath}`);
+          await browser.get(`${serverBasePath}#${opts.redirectPath}`);
```

Comment:
We can probably use the `navigateToApp` function here to be a bit more consistent with other tests. Maybe something like:

```
await PageObjects.common.navigateToApp('kibana', {
  basePath: spaceId === 'default' ? `/` : `/s/${spaceId}`,
});
```

Reply:
In hindsight, I'm realizing adding `switchToSpace` is overkill since you can just hit a space using the URL rather than clicking the UI to navigate between spaces every time. Pushing an update to remove this entirely and just use a modified `navigateToApp`.

Thanks for the input!